cmd/mock-onelogin/main.go

```diff
@@ -296,3 +357,40 @@ func logRoute(h http.Handler) http.HandlerFunc {
 		h.ServeHTTP(w, r)
 	}
 }
+
+func userDetails(key string) (givenName, familyName, birthDate string) {
+	switch key {
+	case "donor":
+		return "Sam", "Smith", "2000-01-02"
+	case "attorney":
+		return "Amy", "Adams", "1980-01-02"
+	case "certificate-provider":
+		return "Charlie", "Cooper", "1990-01-02"
```

Comment:
I think it would be a bit clearer to key these as e.g. `sam-smith` rather than `donor`, since there's nothing explicitly donor-y about the details (I assume they just happen to match up in Use)?

Reply:
Leaving this in for now to be in line with make's implementation. To be reviewed at some point in the future